index.js

```diff
@@ -0,0 +1,56 @@
+window.document.querySelector('#note-form').addEventListener('submit', function(event) {
+    event.preventDefault(); 
+
+    const title = window.document.querySelector("#note-title").value;
```

Comment:
It was suggested to use `event` to extract target values.

- [ ] Implement `event.target`.